tests/e2e/page/RtSocial.js

```diff
@@ -0,0 +1,171 @@
+        await this.page.goto("./wp-admin/options-general.php?page=rtsocial-options", { waitUntil: "load" });
+    }
+    // this function is used to validate all the visible options in the settings page
+    async validateVisibleOptions(){
+        const placementSettings = this.page.locator(selectors.placementSettingsDiv)
+        expect(placementSettings).not.toBeNull();
+        const buttonStyle = this.page.locator(selectors.buttonStyleDiv)
+        expect(buttonStyle).not.toBeNull();
+        const alignmentSetting = this.page.locator(selectors.leftAlignment);
+        expect(alignmentSetting).not.toBeNull();
+        const activeButtons = this.page.locator(selectors.activeButtonDiv);
+        expect(activeButtons).not.toBeNull();
+        const twitterHandle = this.page.locator(selectors.twitterHandleInput);
+        expect(twitterHandle).not.toBeNull();
+        const facebookToken = this.page.locator(selectors.faceBooktokenInput);
+        expect(facebookToken).not.toBeNull();
+    }
+    // this function is used to test Alignment Settings checkboxes
+    async validateAlignmentCheckbox(){
+        expect(selectors.leftAlignment).not.toBeNull()
+        expect(selectors.rightAlignment).not.toBeNull()
+        expect(selectors.centerAlignment).not.toBeNull()
+        expect(selectors.noneAlignment).not.toBeNull()
+    }
+    // this function is used to Test Button Style checkboxes
+    async validateButtonStyleCheckbox(){
+        expect(selectors.horizontalButtonStyle).not.toBeNull();
+        expect(selectors.iconCountButtonStyle).not.toBeNull();
+        expect(selectors.iconButtonStyle).not.toBeNull();
+        expect(selectors.verticalButtonStyle).not.toBeNull();
+    }
+    // this function is used to Test Placement Checkboxes
+    async validatePlacementCheckbox(){
+        expect(selectors.topPlacement).not.toBeNull();
+        expect(selectors.bottomPlacement).not.toBeNull();
+    }
+
+    // this function is used to set Left aligned in top position with icon style Social Icons
+    async setLeftAlignTopPlaceIcon(){ 
+        await this.page.locator(selectors.topPlacement).check();
+        await this.page.locator(selectors.iconButtonStyle).check()
+        await this.page.locator(selectors.leftAlignment).check();
+    }
+    // this functions is used to save settings
+    async saveSetting(){
+        await this.page.locator(selectors.buttonSaveSetting).click();
+        expect(this.page.locator(selectors.messageSaveSetting)).not.toBeNull();
+    }
+    // this function is used to validate Left aligned in top position with iconCount style Social Icons
+    async validateLeftAlignTopPlaceIcon(){
+        await Promise.all([
+            this.page.click(selectors.adminBar),
+        ]);
+        await expect(this.page.locator(selectors.leftAlignedValidate).first()).toBeVisible();
+    }
+    // this function is used to set Right aligned in top position with iconCount style Social Icons
+    async setRightAlignTopPlaceIcon(){
```

Comment:
@alvitazwar 
Add assertion here to validate whether the element is a checkbox for the right alignment is checked.
`expect(this.page.locator(selectors.rightAlignment)).toBeChecked();`